core/src/test/java/org/infinispan/stream/DistributedSequentialNonRehashStreamTest.java

```diff
@@ -10,7 +10,7 @@
  * Verifies stream tests work when rehash is disabled on a sequential stream
  */
 @Test(groups = "functional", testName = "streams.DistributedSequentialNonRehashStreamTest")
-@InCacheMode({ CacheMode.DIST_SYNC, CacheMode.SCATTERED_SYNC })
+@InCacheMode({ /*CacheMode.DIST_SYNC,*/ CacheMode.SCATTERED_SYNC })
```

Comment:
Temporary?

Reply:
I am not sure how this got in here.... I didn't even look at that class, lol.